### DISCUSSION TITLE
A white stripe in the title.

### DISCUSSION THREAD

**Discussion by AADJND5** [sumatrapdfreader/sumatrapdf#4292] [General]
Why is this area not black like in previous versions? What is the meaning of the white stripe, do as before.
![346](https://github.com/sumatrapdfreader/sumatrapdf/assets/113119669/7ba7bc7c-ac1c-4b99-a33b-0041996c92a9)
![352](https://github.com/sumatrapdfreader/sumatrapdf/assets/113119669/1fbe54f1-7c23-4014-8b48-01ddf4370180)

the snapshot №1  Sumatra PDF 3.4.6
the snapshot №2  Sumatra PDF 3.5.2

**Comment by GitHubRulesOK** [comment]
Side comment why is your logo fuzzy that does not look official for support?

There were many changes in the title/tabs area some cause different behaviours in different versions with different settings so it is frequently affected by changes that may even look different for different users on same machine or different windows versions.

Default header will be clean, mean and light.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/98a99bbb-2a25-48a1-8b1e-c7afbcf55cac)
Windows System colours will colour the title bar
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/fb245597-9ca2-433f-9c91-51431070dac8)
Dark background will look better by contrast with Windows selection, but SumatraPDF generally cant tell what colour windows is using.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/29e2e098-5960-4ca4-abcb-37a8bdb6b243)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/72913582-d190-479b-8f11-3dba144d5549)

**Comment by AADJND5** [comment]
3.5.2 original
![3 5 2 оригинал](https://github.com/sumatrapdfreader/sumatrapdf/assets/113119669/a376ea9e-a587-4246-806c-f0569eed0b23)

**Comment by AADJND5** [comment]
No other programs are installed. The icons on the panel are slightly distorted. My Windows 10 Pro system,

**Comment by AADJND5** [comment]
I see, thanks for the clarification. I will stay on the Sumatra PDF version 3.4.6 for now. Thanks for the program. I've been with you for many years.

**Reply by GitHubRulesOK** [reply]
you can see the many changes including improved icon handling in the 16309 pre-release I show as current status on Windows 10 Pro
Icons are better but there is still only light or dark for tabs area which is not the system colour.

**Comment by AADJND5** [comment]
Yes, indeed, the icons are clearly drawn in beta 3.6, but I will stay on version 3.4.6 for now, it has never let me down, I am very pleased with your program.
![3 6](https://github.com/sumatrapdfreader/sumatrapdf/assets/113119669/2a58215d-2619-495c-b8f1-2cde6caad49a)

**Reply by GitHubRulesOK** [reply]
PS not my app I simply support it :-)
